client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala

```diff
@@ -173,7 +173,7 @@ class LifecycleManager(val appUniqueId: String, val conf: CelebornConf) extends
   if (authEnabled) {
     logInfo(s"Authentication is enabled; setting up master and worker RPC environments")
     val appSecret = createSecret()
-    applicationMeta = ApplicationMeta(appUniqueId, appSecret)
+    applicationAuthMeta = ApplicationMeta(appUniqueId, appSecret)
```

Comment:
Add else to initialize ApplicationMeta when auth disabled?

Reply:
thanks, addressed.